### ISSUE TITLE
Deck-Board user interface does not react as expected when deleting a card

### ISSUE THREAD

**Issue by aixzellent** [nextcloud/deck#2429]
Deck-Board user interface does not react as expected when deleting a card. Deleted cards will be removed from the view only after you manually check the website again.

**To Reproduce**
1. Go to 'Deck (e.g. personal board)'
2. Click on 'delete card (right click the ...-menu)'
3. nothing happens

**Expected behavior**
After an (optional) query, the card must be removed from the board.

**Screenshots**
-

**Client details:**
 - OS: Ubuntu 18.04
 - Browser: chromium, chrome, firefox... does'nt matter
 - Version: 84.0.4147.135 does'nt matter
 - Device: desktop

<details>
<summary>Server details</summary>
**Operating system**: Ubuntu 18.04

**Web server:** apache 2

**Database:** mysql

**PHP version:** 7.2

**Nextcloud version:** 19.0.4

**Where did you install Nextcloud from:** nextcloud.com/download

**Signing status:** No errors have been found.
**List of activated apps:**

Enabled:
  - accessibility: 1.5.0
  - activity: 2.12.1
  - bbb: 1.1.0
  - bruteforcesettings: 2.0.1
  - cloud_federation_api: 1.2.0
  - comments: 1.9.0
  - contacts: 3.4.0
  - contactsinteraction: 1.0.0
  - dav: 1.15.0
  - deck: 1.1.0
  - federatedfilesharing: 1.9.0
  - federation: 1.9.0
  - files: 1.14.0
  - files_accesscontrol: 1.9.1
  - files_fulltextsearch: 1.4.3
  - files_fulltextsearch_tesseract: 1.4.2
  - files_mindmap: 0.0.23
  - files_pdfviewer: 1.8.0
  - files_rightclick: 0.16.0
  - files_sharing: 1.11.0
  - files_trashbin: 1.9.0
  - files_versions: 1.12.0
  - files_videoplayer: 1.8.0
  - firstrunwizard: 2.8.0
  - fulltextsearch: 1.4.2
  - fulltextsearch_elasticsearch: 1.5.2
  - logreader: 2.4.0
  - lookup_server_connector: 1.7.0
  - nextcloud_announcements: 1.8.0
  - notifications: 2.7.0
  - oauth2: 1.7.0
  - onlyoffice: 6.0.2
  - password_policy: 1.9.1
  - polls: 1.5.4
  - privacy: 1.3.0
  - provisioning_api: 1.9.0
  - ransomware_protection: 1.7.0
  - recommendations: 0.7.0
  - serverinfo: 1.9.0
  - settings: 1.1.0
  - sharebymail: 1.9.0
  - spreed: 9.0.4
  - support: 1.2.1
  - survey_client: 1.7.0
  - systemtags: 1.9.0
  - text: 3.0.1
  - theming: 1.10.0
  - twofactor_backupcodes: 1.8.0
  - updatenotification: 1.9.0
  - viewer: 1.3.0
  - workflowengine: 2.1.0
Disabled:
  - admin_audit
  - calendar
  - encryption
  - files_external
  - mail
  - photos
  - user_ldap

**Nextcloud configuration:**

{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "dms.tcaix.de"
        ],
        "default_language": "de",
        "default_locale": "de_DE",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "19.0.4.2",
        "overwrite.cli.url": "https:\/\/dms.tcaix.de\/nextcloud",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "onlyoffice": {
            "verify_peer_off": true
        },
        "maintenance": false,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpsecure": "tls",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "theme": "",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}

**Are you using an external user-backend, if yes which one:** no

**Comment by cyberswan** [comment]
Hello,m
I have the same behavior, also when I archive a card.

**Comment by simonjaq** [comment]
Hello

For me the exact same behaviour. But not only on deleting cards but also creating new cards, assigning users, comments etc. Nothing happens until I refresh the browser window.

**Comment by aixzellent** [comment]
This fix worked for me: 
Properly check if FTSEvent has an argument set #2358
https://github.com/nextcloud/deck/pull/2358/files

**Comment by simonjaq** [comment]
Ah yes. Updating to 1.1.1 fixed it. All working well now.

**Comment by juliushaertl** [comment]
Closing as a duplicate of #2344 then :wink:
